spec/features/admin/configuration/states_spec.rb

```diff
@@ -11,7 +11,7 @@
   before(:each) do
     login_as_admin
     @hungary = Spree::Country.create!(name: "Hungary", iso_name: "Hungary")
-    Spree::Config[:default_country_id] = country.id
+    DefaultCountry.code = country.iso
```

Comment:
So if we drop the setter here, we just need the default country to be Hungary for the duration of this test, right? So we could do something like:
```ruby
@hungary = Spree::Country.create!(name: "Hungary", iso_name: "Hungary", iso: "HU")

allow(ENV).to receive(:[]).and_call_original
allow(ENV).to receive(:[]).with("DEFAULT_COUNTRY_CODE").and_return("HU")
```

Reply:
I think that looks 💯 :)